odoo/service/server.py

```diff
@@ -265,6 +265,8 @@ def dispatch(self, event):
         if isinstance(event, (FileCreatedEvent, FileModifiedEvent, FileMovedEvent)):
             if not event.is_directory:
                 path = getattr(event, 'dest_path', event.src_path)
```

Comment:
We can rewrite it as a single-line diff.
```suggestion
                path = getattr(event, 'dest_path', '') or event.src_path
```
I am not aware of any employee running Win11 so it is going to be a bit hard to test the diff our side. Pinging @Icallhimtest who seems to be knowledgeable with this class.

Reply:
@lcallhimtest @Julien00859 watchdog package version is 4.0.0, there is also the same problem on macOS Sonoma.

Reply:
No issue for me on this change :+1: